test_scripts/RC/MultipleModules/commonRCMulModules.lua

```diff
@@ -507,7 +507,8 @@ local function getInfo(pModuleType, pModuleId, pInfoType)
   }
 
   if pInfoType == "INCORRECT_MODULE_TYPE" then
-    return "Ignored invalid value"
+    return "RPC.msg_params.moduleType: Ignored invalid value - " .. pModuleType
+      .."\nRPC.msg_params: Missing mandatory parameter: moduleType"
```

Comment:
We should not match against the exact string, that leads to fragile tests as we can see by this breakage. At this point I might suggest just removing the info check entirely.

Reply:
@jacobkeeler I would propose more elegant solution. Fortunately `string.find` function has an option of 'plain' comparison. Please find implementation in 5cf693f.